game.py

```diff
@@ -0,0 +1,364 @@
+from deck import Deck, NoMoreCards
+from player import Player
+from card import Card, NormalCard, SpecialCard, CardType
+
+class Game:
+    def __init__(self, gdeck):
+        self.players = []
+        self.number_of_players = len(self.players)
+        self.gdeck = Deck(gdeck)
+        self.gdeck.shuffle_deck()
+        self.round = 0
+        self.current_round_color = None
+        self.cards_played = []
+        self.pace = 1
+        self.skip_player = False
+    
+    def start_game(self):
+        while True:
+            if self.player_won_game():
+                break
+                    
+            self.single_round()
+    
+    def single_round(self):
```

Comment:
Take a look on shape of this function (https://www.fluentcpp.com/2020/01/14/the-shapes-of-code/)
You should move some parts of this function to another functions to make it clear what's going on here on each step :)